scalding-repl/src/main/scala/com/twitter/scalding/ScaldingILoop.scala

```diff
@@ -85,14 +89,15 @@ class ScaldingILoop
 
   override def createInterpreter() {
     super.createInterpreter()
-    addThunk {
-      intp.beQuietDuring {
-        addImports(imports: _*)
+    intp.beQuietDuring {
+      addImports(imports: _*)
 
-        // interpret all files named ".scalding_repl" from the current directory up to the root
-        findAllUpPath(".scalding_repl")
-          .reverse // work down from top level file to more specific ones
```

Comment:
Hmm. I guess reversing the list guarantees that local settings override higher-up settings? So maybe we should add back the reverse.